pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -35,16 +32,6 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 			Labels:    c.getLabels(mon, false, true),
 		},
 		Spec: v1.ServiceSpec{
-			Ports: []v1.ServicePort{
-				{
-					Name: "tcp-msgr1",
-					Port: mon.Port,
-					// --public-bind-addr=IP with no IP:port has the mon listen on port 6789
-					// regardless of what port the mon advertises (--public-addr) to the outside.
-					TargetPort: intstr.FromInt(int(DefaultMsgr1Port)),
-					Protocol:   v1.ProtocolTCP,
-				},
-			},
```

Comment:
Is it still safe to remove this when we have msgr1 supported still?

Reply:
This block is basically replaced below on line 45 where the msgr1 port is added as long as msgr2 isn't required.